### DISCUSSION TITLE
EdgeDB 1.0 Beta 3

### DISCUSSION THREAD

**Discussion by 1st1** [geldata/gel#2820] [General]
![image](https://user-images.githubusercontent.com/239003/129657572-2f51eb6d-210a-461b-8f9a-9698becfe349.png)

On August 12th 2021 we released EdgeDB Beta 3 "Ross". Read the announcement blog post here: https://www.edgedb.com/blog/edgedb-beta-3-ross

**Comment by CodesInChaos** [comment]
I think `relative_duration` could use some additional documentation due to the (unavoidable) un-intuitive behaviour when interacting with dates:

1.  Document the order in which adding/subtracting a relative_duration to/from a date goes through the three components, since that affects the result
2. Document how addition handles cases where the resulting target day-of-month or hour-of-day doesn't exist.
3. Call out that it doesn't follow the associative law, i.e. `(date + rel1) + rel2 != date + (rel1 + rel2)` in some cases
4. Call out that subtraction isn't the inverse of addition, i.e. `(date + rel) - rel != date` in some cases
